application-admintools-api/pom.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+-->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>application-admintools</artifactId>
+        <groupId>com.xwiki.admintools</groupId>
+        <version>1.0-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>application-admintools-api</artifactId>
+    <packaging>jar</packaging>
+    <name>Admin Tools(Pro) - API</name>
+    <description>Provides the Java APIs needed by the Admin tools Application.</description>
+    <properties>
+        <maven.compiler.source>11</maven.compiler.source>
+        <maven.compiler.target>11</maven.compiler.target>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
```

Comment:
Why are these needed? (I see them in all your poms)

Reply:
Needed or not they definitely don't need to be in *all* the poms for sure, you should put what you need in your root pom.

Reply:
They were generated when I created the projects and I thought they are needed.
I removed them and it works fine.

Reply:
I created blank maven projects and then manually edited each pom

Reply:
> created blank maven projects

I mean, which tool did you use to do that that generated those properties.

Reply:
I used Intellij 2022.22.1 community edition
Here is a screenshot of an example of how I configured it:
![image](https://github.com/xwikisas/application-admintools/assets/104432589/2f601e8d-f4e1-4cec-896d-66f0ace7609c)

Reply:
OK so no something on our side we could fix. I guess IntelliJ added those properties because of that `JDK` field, and you indicated Java 11.